### DISCUSSION TITLE
Is it possible to have 2 images next to each other, using columns for example?

### DISCUSSION THREAD

**Discussion by notflip** [nuxt/content#2113] [Q&A] [answered]
I'm wondering if there's a way to have 2 or 3 columns in markdown, for example to put images next to each other?

**Comment by nobkd** [comment, marked as answer]
Sure. You can create a block component that is styled as grid and use multiple images inside.
E.g.
```md
<!-- some markdown file -->
::cols

![](imglink1)
![](imglink2)

::
```

```vue
<!-- components/content/Cols.ts -->
<template>
  <div>
    <ContentSlot :use="$slots.default" unwrap="p" />
  </div>
</template>

<style scoped>
div {
  display: grid;
  grid-template-columns: repeat(2, 1fr);
}
:deep(img) {
  max-width: 100%;
}
</style>
```
This would be fixed for two columns.

The `:deep(img)` is there to prevent images from overflowing, 'cause they seem to get the full site width in the default setting... Maybe you don't need this rule (with some style resets)... 🤷
